README.md

```diff
@@ -4,317 +4,101 @@
-
-```
-rules_version = '2';
-
-service cloud.firestore {
-  match /databases/{database}/documents {
-      // Administrator role
-      match /Events/{document=**} {
-        allow read: if request.auth != null && request.auth.token.admin == true;
-      }
-      match /Donations/{document=**} {
-        allow read: if request.auth != null && request.auth.token.admin == true;
-      }
-      match /DonationDetails/{document=**} {
-        allow read: if request.auth != null && request.auth.token.admin == true;
-      }
-      match /Images/{document=**} {
-        allow read: if request.auth != null && request.auth.token.admin == true;
-        }
-        match /ImageDetails/{document=**} {
-        allow read: if request.auth != null && request.auth.token.admin == true;
-        }
-      match /Organizations/{document=**} {
-        allow create, read, update, write: if request.auth != null && request.auth.token.admin == true;
-        }
-      match /Storage/{document=**} {
-        allow create, read, update, write: if request.auth != null && request.auth.token.admin == true;
-        }
-      match /Users/{document=**} {
-        allow read: if request.auth != null && request.auth.token.admin == true;
-      }
-      match /UserDetails/{document=**} {
-        allow read, update: if request.auth != null  && request.auth.token.admin == true;
-        }
-      // Standard user role
-        match /Donations/{document=**} {
-        allow create, read, update, write: if request.auth != null;
-        }
-        match /DonationDetails/{document=**} {
-        allow create, read, update, write: if request.auth != null;
-        }
-      match /Images/{document=**} {
-        allow create, read: if request.auth != null;
-        }
-        match /ImageDetails/{document=**} {
-        allow create, read: if request.auth != null;
-        }
-        match /Users/{userId} {
-        allow create, read, update, write: if request.auth != null && request.auth.uid == userId;
-        }
-        match /UserDetails/{userId} {
-        allow create, read, update, write: if request.auth != null && request.auth.uid == userId;
-        }
-  }
-}
-```
-
-**********\*\*\*\***********\*\***********\*\*\*\***********firestore.indexes.json**********\*\*\*\***********\*\***********\*\*\*\*********** — (Same directory as above) Database index definitions the emulator should have set [Manage indexes in Cloud Firestore  |  Firebase (google.com)](https://firebase.google.com/docs/firestore/query-data/indexing)
-
+export FIREBASE_CONFIG="$(cat /home/user/projects/baby-equipment-exchange/firebaseConfig.json)"
+export NEXT_PUBLIC_FIREBASE_CONFIG="$(cat /home/user/projects/baby-equipment-exchange/firebaseConfig.json)"
+npm run dev
 ```
-{
-    "indexes": [],
-    "fieldOverrides": []
-}
+13. see the output you can run in your host machine browser http://localhost:3000
+
+
+## Dev Local Setup
+Below are linux command used to setup the environment 
```

Comment:
Add a comment that notes for setup on Mac are in `docs\setup_notes.md`, or a short list of the Mac commands (like `brew`) here.

Reply:
i'm going to ask chatgpt to give the equivalent commands for mac using brew , and i will put that in the read me

Reply:
or since i can't verify it , ill just say that notes for setup on Mac are in docs\setup_notes.md